swift/apple/FirezoneKit/Sources/FirezoneKit/Features/SettingsView.swift

```diff
@@ -26,12 +26,12 @@ public final class SettingsViewModel: ObservableObject {
   public init(tunnelStore: TunnelStore, logger: AppLogger) {
     self.tunnelStore = tunnelStore
     self.logger = logger
-    settings = Settings.defaultValue
+    self.settings = tunnelStore.settings
```

Comment:
Fixes another exposed race condition that we hit more often now because we connect faster and don't switch between states unless we need to.